test/unit/sorted-views.spec.js

```diff
@@ -391,4 +391,85 @@ describe('collection/composite view sorting', function() {
       });
     });
   });
+
+  describe('when using `{ reorderOnSort: true }`', function () {
+    var texts = {
+      asOption: 'as an option',
+      onPrototype: 'on the prototype',
+      viewComparator: 'with viewComparator'
+    };
+
+    var getSpecTitle = function(options) {
+      return _.map(options, function (v, k) {
+        return texts[k];
+      }).join(' ');
+    };
+
+    var describeSpec = function (specOptions) {
+      describe(getSpecTitle(specOptions), function () {
+        beforeEach(function () {
+          var commonAttrs = {
+            childView: this.ChildView,
+            collection: this.collection
+          };
+
+          if (specOptions.viewComparator) {
+            commonAttrs.viewComparator = 'foo';
+          }
+
+          if (specOptions.asOption) {
+            this.collectionView = new this.CollectionView(_.extend({}, {
+              reorderOnSort: true
+            }, commonAttrs));
+          } else if (specOptions.onPrototype) {
+            var ReorderedCollectionView = this.CollectionView.extend({
+              reorderOnSort: true,
+              onReorder: sinon.spy(),
+              onBeforeReorder: sinon.spy()
```

Comment:
ok. Is there a reason why you do not use the global `sinon` object ? Did you have some problems with it being polluted during the tests ?